spyder/__init__.py

```diff
@@ -48,7 +48,12 @@
 DATAPATH = LOCALEPATH = DOCPATH = MATHJAXPATH = JQUERYPATH = ''
 
 
-import os
+import os, sys
+
+# Fixes launching issues with Big Sur (GH14222)
```

Comment:
Please change `GH14222` for `spyder-ide/spyder#14222` (that's our convention to cite issues in our codebase).